sources/packages/forms/src/form-definitions/reportscholasticstandingchange.json

```diff
@@ -8,6 +8,44 @@
     "common"
   ],
   "components": [
+    {
+      "input": true,
+      "tableView": true,
+      "key": "applicationOfferingUnformattedStartDate",
+      "label": "applicationOfferingUnformattedStartDate",
+      "protected": false,
+      "unique": false,
+      "persistent": true,
```

Comment:
It does not need to be persistent unless I am missing something. Same for the other field.
![image](https://github.com/user-attachments/assets/74c163ae-5ff0-43d4-a42f-86eb70934f4a)

Reply:
It needs else the dryrun fails in the API